packages/data-context/src/actions/MigrationActions.ts

```diff
@@ -147,6 +168,13 @@ export class MigrationActions {
     })
   }
 
+  locallyInstalledCypressVersion (currentProject: string) {
+    const localCypressPkgJsonPath = resolveFrom(currentProject, path.join('cypress', 'package.json'))
+    const localCypressPkgJson = fs.readJsonSync(path.join(localCypressPkgJsonPath)) as { version: string }
```

Comment:
We should wrap this in a try catch as we won't display the proper error `MIGRATION_CYPRESS_NOT_FOUND` as it will throw when reading a non-existent file.

I tested this by clicking on a project that had no node_modules
<img width="919" alt="Screen Shot 2022-04-29 at 10 01 10 AM" src="https://user-images.githubusercontent.com/25158820/165970971-50fe69cb-968f-4715-8a6d-7f03f6b39543.png">

Reply:
https://github.com/cypress-io/cypress/pull/21255/commits/6c3aa9c787533dbf86853b58e91bb0120c5775e6

Reply:
Done!